test/Maestro/tutorial.yml

```
@@ -0,0 +1,29 @@
+#After registration you'll go through the tutorial screens.
+---
+- tapOn: "Let's go!"
+- tapOn:
+    label: Step 1/6
+    point: "85%,46%" 
+- tapOn:
+    label: Step 2/6
+    point: "85%,81%" 
+- tapOn:
+    label: Click outside box should not do anything
+    point: "55%,55%" 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)